common/djangoapps/student/admin.py

```diff
@@ -267,6 +270,40 @@ def get_readonly_fields(self, request, obj=None):
         return django_readonly
 
 
+class UserResource(resources.ModelResource):
+    """ Resource for importing and exporting User objects. """
+
+    def before_save_instance(self, instance, using_transactions, dry_run):
+        """
+        Add generated email and set password for User instance.
+        """
+        if not instance.email:
+            instance.email = instance.username + '@ex.ex'
+        instance.set_password(instance.password)
+
+    def after_save_instance(self, instance, using_transactions, dry_run):
+        """
+        ACT specific creation of UserProfile with additional params.
+        """
+        UserProfile.objects.create(user=instance, is_anon=True)
+
+    class Meta:
+        model = User
+        add_fields = tuple(settings.USER_BULK_REGISTRATION_IMPORT_EXPORT_FIELDS) \
+         if hasattr(settings, 'USER_BULK_REGISTRATION_IMPORT_EXPORT_FIELDS')  else ()
```

Comment:
`USER_BULK_REGISTRATION_IMPORT_EXPORT_FIELDS` has default value (set in aws.py => () ), why do we need this `if` check?

Reply:
As it exist only in Raccoon Gang section of settings, it might be not set in aws.py in other projects, but this code will be still useful.